enroll/transport_http.go

```diff
@@ -61,8 +62,18 @@ func decodeMDMEnrollRequest(_ context.Context, r *http.Request) (interface{}, er
 		if err != nil {
 			return nil, err
 		}
-		// TODO: We should verify but not currently possible. Apple
-		// does no provide a cert for the CA.
+		err = p7.Verify()
```

Comment:
Until we have a solution for #86 then I'm not sure we should respond with 401. That will prompt the end-user to login to the MDM when in fact we should be rejecting the connection. _That said_ we should probably respond with a `4xx` error (maybe `400`? )as it is indeed a client-side/request issue.